protocol/requests/toggle_collapsed_community_category.go

```diff
@@ -0,0 +1,26 @@
+package requests
+
+import (
+	"errors"
+)
+
+var ErrToggleCollapsedCommunityCategoryInvalidCommunityID = errors.New("create-community-category: invalid community id")
+var ErrToggleCollapsedCommunityCategoryInvalidName = errors.New("create-community-category: invalid category name")
```

Comment:
The error says `create-community-category: ...`, shouldn't it be something like `toggle-collapsed-community-category: invalid category name`? Same thing for the error above it.